app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteAddFragment.kt

```diff
@@ -36,6 +37,16 @@ class AutocompleteAddFragment : Fragment() {
         updater.updateIcon(R.drawable.ic_close)
     }
 
+    override fun onPause() {
```

Comment:
I wonder why you chose onPause() and not for example saveDomainAndClose(). Doesn't using onPause() mean we will also dismiss the keyboard if the user leaves the app and returns back to it?

Reply:
Hi @pocmo 
I used onPause() to ensure the keyboard is hidden even when the user decides not to add a domain or search engine. 
Example:

* go to Settings > URL Autocomplete > Custom URLs > Add custom URL
* Enter a random URL (not required)
* Press "Cancel" or X

If the code to hide the keyboard is located in saveDomainAndClose() the keyboard will remain open until closed by the user.

I tested the changes I made by closing and re-opening the app and the code in onPause() does not seem to have any effect although I understand that it is an unnecessary call when the app closes

Reply:
I see. This makes sense. Thank you for testing this.